pkg/build/build.go

```diff
@@ -88,9 +88,7 @@ func Build(manifest model.Manifest) error {
 		return fmt.Errorf("failed to package license file: %v", err)
 	}
 
-	if manifest.DockerOutput == model.DockerOutputContext {
-		log.Warnf("Docker output in 'context' mode; will not produce SBOM.")
-	} else if manifest.SkipGenerateBillOfMaterials {
+	if manifest.SkipGenerateBillOfMaterials {
```

Comment:
We create a new option, SkipGenerateBillOfMaterials, and we output a message if set, but I don't see that we actually act on it. I would have expected some check around the actual BOM generation to not run if this value is set.